packages/actions-global/src/actions/handle-action-error.ts

```diff
@@ -9,15 +9,19 @@ import browserHistory from '@commercetools-frontend/browser-history';
 import showApiErrorNotification from './show-api-error-notification';
 import showUnexpectedErrorNotification from './show-unexpected-error-notification';
 
-type ActionError =
-  | Error
-  | {
-      statusCode: TStatusCode;
-      body: {
-        message: string;
-        errors?: TAppNotificationApiError | TAppNotificationApiError[];
-      };
-    };
+type SdkError = {
```

Comment:
Nitpick: not 100% sure to name this `SdkError`, maybe it’s more like `ApiError`

Reply:
Absolutely. Overlooked that we are using term ApiError in all other places.